core/src/test/scala/org/apache/spark/sql/catalyst/plans/BasePlanTest.scala

```diff
@@ -32,24 +32,38 @@ class BasePlanTest extends BaseTiSparkTest {
     case plan: ColumnarCoprocessorRDD => plan
     case plan: ColumnarRegionTaskExec => plan
   }
-  val extractDAGRequest: PartialFunction[SparkPlan, TiDAGRequest] = {
-    case plan: ColumnarRegionTaskExec => plan.dagRequest
-    case plan: ColumnarCoprocessorRDD => plan.dagRequest
+  val extractDAGRequest: PartialFunction[SparkPlan, Seq[TiDAGRequest]] = {
```

Comment:
Why change `TiDAGRequest` to `Seq[TiDAGRequest]`?

Reply:
Are there any problems with the explanation result before without `Seq[]`?

Reply:
> Are there any problems with the explanation result before without `Seq[]`?

It only can get one DAG from multiple TiDAGRequest.